### PR TITLE
Fix up custom op tutorials

### PR DESCRIPTION
**Pull request by tugsbayasgalan**
Fixes #ISSUE_NUMBER

## Checklist

- [ ] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [ ] Only one issue is addressed in this pull request
- [ ] Labels from the issue that this PR is fixing are added to this pull request
- [ ] No unnecessary issues are included into this pull request.

### CODE REVIEW

File: intermediate_source/_torch_export_nightly_tutorial.py [hunk 1]
```diff
@@ -435,34 +435,45 @@ def suggested_fixes():
...
-@impl(m, "custom_op", "Meta")
-def custom_op_meta(x):
+@torch.library.register_fake("mylib::sin")
+def _(x):
     return torch.empty_like(x)
 
+# - Sometimes, the custom op you are exporting has data-dependent output, meaning
+# we can't determine the shape of the output at compile time. In this case, you can do
+# following:
+@torch.library.register_fake("mylib::op_that_has_data_dependent")
```
Comment (ydwu4) [inline intermediate_source/_torch_export_nightly_tutorial.py:459]: Any example is using this fake op op_that_has_data_dependent? Might worth adding one if not.

**Review by ydwu4 — APPROVED** [review]

**Review by avikchaudhuri — APPROVED** [review]

File: intermediate_source/_torch_export_nightly_tutorial.py [hunk 2]
```diff
@@ -435,36 +435,54 @@ def suggested_fixes():
...
+# following:
+@torch.library.custom_op("mylib::nonzero", mutates_args=())
+def nonzero(x):
+    x_np = x.cpu().numpy()
+    res = np.stack(np.nonzero(x_np), axis=1)
+    return torch.tensor(res, device=x.device)
+
+@torch.library.register_fake("mylib::nonzero")
+def _(x):
+    # Number of nonzero-elements is data-dependent.
```
Comment (svekars) [inline intermediate_source/_torch_export_nightly_tutorial.py:467]:
```suggestion
    # The number of nonzero-elements is data-dependent.
```

**Review by svekars — APPROVED** [review]
